client/test/utils/specs_dispatcher.js

```diff
@@ -6,14 +6,12 @@ TestDispatcher.prototype.register = function register(callback) {
   this._callbacks.push(callback);
 };
 
-
 TestDispatcher.prototype.dispatch = function dispatch(payload) {
   this._callbacks.forEach((callback) => {
     callback.call(this, payload);
   });
 };
 
-TestDispatcher.prototype.waitFor = function waitFor(events) {
-};
+TestDispatcher.prototype.waitFor = function waitFor() {};
```

Comment:
Shouldn't we add a real version of the `waitFor` method instead of fakingly mocking it? Does tests pas correctly even?